src/css/Tooltip.scss

```diff
@@ -22,6 +23,11 @@
     opacity: 0;
     transition: opacity 0.25s;
     transition-delay: 0.5s;
+
+    @include mobile {
+        left: 0;
+        margin-left: 0;
+    }
```

Comment:
I'm not sure this works everywhere? For example we have the tooltips in the splits editor variables tab as well.

Reply:
I tested those and it seemed fine